proto/cosmos/authz/v1beta1/authz.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package cosmos.authz.v1beta1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/authz/types";
+
+// SendAuthorization allows the grantee to spend up to spend_limit coins from
+// the granter's account.
+message SendAuthorization {
+  option (gogoproto.equal)                   = true;
+  option (gogoproto.goproto_getters)         = false;
```

Comment:
Can we limit our usage of gogoproto extensions in new code?

Reply:
+1 for keeping default proto getters.

If there are places where it conflicts (e.g. MsgExecAuthorized has a `msgs` field, which generated `GetMsgs`, which conflicts with the manually written `GetMsgs`), we should just rename our manual method to something else (in the previous example, e.g. `GetServiceMsgs`)